### PR TITLE
Db/seed data

### PR DESCRIPTION
**Pull request by TripolarHZ**
finished creating dummy data for shipping and billing addresses, also made changes to billing address, shipping address, and user file.

### CODE REVIEW

File: db/models/users.js [hunk 1]
```diff
@@ -1,10 +1,18 @@
 const bcrypt = require('bcrypt');
 const client = require("../client");
+const { addBillingAddressToUser, getBillingAddressByUserId } = require('./billingAddress');
+const { addShippingAddressToUser, getShippingAddressByUserId } = require('./shippingAddress');
 
 // database functions
 
 // user functions
-async function createUser({ name, email, password, role }) {
+async function createUser({
```
Comment (cmceachin5888) [inline db/models/users.js:9]: I made some edits to users as well. i don't think adding billing/shipping info at creation is required. the users would add this info later to their profile or at checkout.

File: db/models/products.js [hunk 2]
```diff
@@ -97,6 +97,7 @@ async function getAllProducts() {
             SELECT *
             FROM products;
         `);
+        return products;
```
Comment (cmceachin5888) [inline db/models/products.js:100]: i had to make some decent edits to products to get the tests to pass.

**Review by cmceachin5888 — CHANGES_REQUESTED** [review]
please review comments. lets discuss if needed.

File: db/models/users.js (hunk 1 above)
Comment (Scarlette-O) [reply db/models/users.js:9]: Camron is right. Billing and shipping would be added to the profile i don't think its necessary when you create a user.

**Review by Scarlette-O — COMMENTED** [review]
As soon as you make the changes that Cameron requested i will add my approve.

**Review by cmceachin5888 — APPROVED** [review]

**Review by Scarlette-O — APPROVED** [review]
looks good!

**Pull request merged**